tests/test_store_api.py

```diff
@@ -2,4 +2,4 @@
 
 
 def test_version():
-    assert __version__ == "1.0.1"
+    assert __version__ == "1.0.2"
```

Comment:
Is this a test that needs manually updating every time we update the module? I don't think that's a great idea.

Reply:
If you want to test the version, you should simply make sure [semver](https://pypi.org/project/semver/) is able to parse it, maybe? That might actually be useful.

Reply:
This is a good idea. I tried to use `semver.VersionInfo.isvalid()`: https://python-semver.readthedocs.io/en/latest/usage.html#checking-for-a-valid-semver-version

But, I get:
```
AttributeError: type object 'VersionInfo' has no attribute 'isvalid'
```

So I am using the parse as you suggested.